contract/src/otcDesk.js

```diff
@@ -41,14 +36,8 @@ const start = async zcf => {
     makeRemoveInventoryInvitation: () => {
       /** @type OfferHandler */
       const removeInventory = seat => {
-        trade(
-          zcf,
-          {
-            seat: marketMakerSeat,
-            gains: {},
-          },
-          { seat, gains: seat.getProposal().want },
-        );
+        seat.incrementBy(marketMakerSeat.decrementBy(seat.getProposal().want));
```

Comment:
What i suggested (I don't know where the comment disappeared to) was that it's a little hard to see what `.want` is connected to here.  If that's pulled out, I think this is easier to read.
```suggestion
     const { want: seatWant } = seat.getProposal();
      const removeInventory = seat => {
          seat.incrementBy(marketMakerSeat.decrementBy(seatWant));
```